tests/app/test_validation.py

```diff
@@ -489,7 +487,12 @@ def check_schema(schema):
     try:
         validate({}, schema)
     except SchemaError as ex:
-        print('Invalid JSON schema: %s' % ex.message)
+        try:
+            msg = ex.message
+        except AttributeError:
+            msg = str(ex)
+
+        print('Invalid JSON schema: %s' % msg)
```

Comment:
should use ex object straightaway : print('Invalid JSON scheme: %s' % str(ex)
'

Reply:
This looks fine to me for now.  We can think about changing it in the future.